source/game/field/CourseColMgr.cc

```diff
@@ -23,7 +23,7 @@ void CourseColMgr::scaledNarrowScopeLocal(f32 scale, f32 radius, KColData *data,
         data = m_data;
     }
 
-    data->narrowScopeLocal(pos / scale, radius / scale, mask);
+    data->narrowScopeLocal(pos.multInv(scale), radius / scale, mask);
```

Comment:
```suggestion
    f32 invScale = 1.0f / scale;
    data->narrowScopeLocal(pos * invScale, radius * invScale, mask);
```

Reply:
Since you prefer breaking out the `invScale` into its own variable, I've done so for every other use as well. I also realized I missed a couple more functions that need this change.